### PR TITLE
setup Travis

### PR DESCRIPTION
**Pull request by gotcha**
- [x] Travis will need to be setup on collective
- [ ] And coveralls as well

### CODE REVIEW

**Comment by gotcha** [comment]
@mauritsvanrees Tests pass, would you have time to review ?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+cache:
+  pip: true
+  directories:
+    - $HOME/buildout-cache
+before_install:
+  - mkdir -p $HOME/buildout-cache/{eggs,downloads}
+  - mkdir $HOME/.buildout
+  - echo "[buildout]" > $HOME/.buildout/default.cfg
+  - echo "download-cache = $HOME/buildout-cache/downloads" >> $HOME/.buildout/default.cfg
+  - echo "eggs-directory = $HOME/buildout-cache/eggs" >> $HOME/.buildout/default.cfg
```
Comment (mauritsvanrees) [inline .travis.yml]: It is easier to set `cache: directories: eggs`, like `buildout.plonetest` does. Then you can skip the verbose creation `~./buildout/default.cfg` and the directories.
But your version works too, so I don't mind.

File: collective/noindexing/testing.py [hunk 2]
```diff
@@ -1,22 +1,27 @@
+from plone.testing import Layer
 from plone.app.testing import IntegrationTesting
-from plone.app.testing import PLONE_FIXTURE
-from plone.app.testing import PloneSandboxLayer
-from zope.configuration import xmlconfig
+from plone.app.testing import PloneWithPackageLayer
```
Comment (mauritsvanrees) [inline collective/noindexing/testing.py]: I have not noticed `PloneWithPackageLayer` before. Looks useful.
Note to self: start using this more.

Comment (gotcha) [reply collective/noindexing/testing.py]: For a reason I do not get, ``PloneWithPackageLayer`` is not used by Timo and friends. Disclaimer: @jfroche and me built ``PloneWithPackageLayer``.

**Comment by gotcha** [comment]
@mauritsvanrees Thanks for merging ! Are you ok that I release ?

**Comment by mauritsvanrees** [comment]
Almost. I made a change to the readme in PR #5. When you review and merge that one, you can make a final release.

**Pull request merged**